rdkit/Chem/Pharm3D/EmbedLib.py

```diff
@@ -822,6 +836,40 @@ def MatchFeatsToMol(mol, featFactory, features):
   return True, res
 
 
+def MatchFeatsToMolPartial(mol, featFactory, pcophore):
+  """ generates a list of all possible mappings of each feature to a molecule,
+  allowing for partial matching of features
+  """
+  explicit = False
```

Comment:
seems clearer to just use: `explicit = isinstance(pcophore, Pharmacophore.ExplicitPharmacophore)`